src/polyfill.ts

```diff
@@ -1,40 +1,40 @@
+
+// @ts-ignore
```

Comment:
why `@ts-ignore` the import ?

Reply:
I don't think so indeed ! Maybe just adding a comment would make this clearer

Reply:
968d8050f744e67f8a8167253f2d67febbadda51